<launch>

<!-- #############    Localization  ##############-->
<!--node pkg="localization" type="encoder_odom_publisher" name="encoder_odom_publisher" /-->  
<!-- This basic robot package locks the map and odom frames -->
<!-- Comment out if another node will provide map to odom transform -->
<node pkg="tf" type="static_transform_publisher" name="tf_map_odom" args="0 0 0 0 0 0 map odom 10" /> 
<node pkg="localization" type="tf_pub" name="tf_odom_baselink" /> 
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find veronica)/veronicaRviz.rviz" />
<node pkg="localization" type="rviz_click_to_2d" name="rviz_click_to_2d" />

 <!-- Run the map server --> 
    <node name="map_server" pkg="map_server" type="map_server" args="$(find veronica)/maps/course_map.pgm .1"/>

 <!-- <include file="$(find veronica)/launch/sensors.launch" /> -->
 <include file="$(find veronica)/launch/manual_drive.launch" />

 

 <!--- Run AMCL --> 
    <!--include file="$(find amcl)/examples/amcl_omni.launch" /-->

   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <remap from="odom" to="encoder/odom_quat" />
    <rosparam file="$(find veronica)/params/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find veronica)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find veronica)/params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find veronica)/params/global_costmap_params.yaml" command="load" /> 
    <rosparam file="$(find veronica)/params/base_local_planner_params.yaml" command="load" />
 </node>


</launch> 